LifeCycleAssessment_Engine/Query/ProjectLifeCycleAssessmentResults.cs

```diff
@@ -0,0 +1,166 @@
+                buildingLifespan = 0;
+            }
+
+            string constructionScope = "";
+            if (lcaResult.LifeCycleAssessmentScope.ConstructionScopeNew == true)
+                constructionScope = "New Construction";
+            else constructionScope = "Renovation";
+
+            string contact = lcaResult.LifeCycleAssessmentScope.ContactName;
+            if (contact == "")
+            {
+                BH.Engine.Reflection.Compute.RecordNote("Please enter your contact information within the LifeCycleAssessmentScope.");
+                contact = "Undefined";
+            }
+
+            string elementScope = "";
+            if(lcaResult.Results.Select(x => x.Scope).Count() > 0)
+            {
+                foreach (var b in lcaResult.Results.ToList())
+                    elementScope += b.Scope.ToString() + ",";
+                elementScope = elementScope.Remove(elementScope.LastIndexOf(','));
+            }
+            else
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("No objects have been provided for evaluation within the discipline-specific scope objects.");
+            }
+
+            string gravitySys = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.GravityStructuralMaterial);
+            if(gravitySys == "Undefined")
+                BH.Engine.Reflection.Compute.RecordNote("Please enter your primary gravity structural system within the LifeCycleAssessmentScope.");
+
+            double gwp = lcaResult.TotalGlobalWarmingPotential;
+            if (gwp <= 0)
+            {
+                BH.Engine.Reflection.Compute.RecordError("No GWP amount was accounted for. Please ensure you have provided objects with EPDs for evaluation before submitting to the central database.");
+                return null;
+            }
+
+            double projectArea = lcaResult.LifeCycleAssessmentScope.ProjectArea;
+            if (projectArea <= 0)
+            {
+                BH.Engine.Reflection.Compute.RecordWarning("Please enter your project's total area within the LifeCycleAssessmentScope.");
+                projectArea = double.NaN;
+            }
+
+            double gwpPerArea = 0;
+            if (double.IsNaN(projectArea))
+            {
+                BH.Engine.Reflection.Compute.RecordError("No area was recorded for this project. The key metric of GWP per area cannot be calculated.");
+                return null;
+            }
+
+            gwpPerArea = gwp / projectArea;
+
+            if (gwpPerArea <= 0)
+                BH.Engine.Reflection.Compute.RecordWarning("Insufficient data was provided to calculate GWP per area. Please provide a project area and valid objects.");
+
+            string lateralSys = System.Convert.ToString(lcaResult.LifeCycleAssessmentScope.LateralStructuralMaterial);
+            if (lateralSys == "Undefined")
```

Comment:
```suggestion
            if (lateralSys.ToLower() == "undefined")
```

As above, suggestion only

Reply:
Same comment as above if that makes sense.